spec/features/publish/courses/new_applications_open.rb

```diff
@@ -0,0 +1,61 @@
+require "rails_helper"
+
+feature "choosing an application open from date" do
+  before do
+    given_i_am_authenticated_as_a_provider_user
+    when_i_visit_the_new_applications_open_from_page
+  end
+
+  scenario "selecting as soon as open on Find" do
+    when_i_select_as_soon_as_open_on_find
+    and_i_click_continue
+    then_i_am_met_with_the_start_date_page
+  end
+
+  # scenario "selecting a custom date" do
```

Comment:
Redundant?

Reply:
Fixed in https://github.com/DFE-Digital/teacher-training-api/pull/2471/commits/60df87b103d58b89e4fce59c4741a2b515c716a2